ArchiSteamFarm.Tests/Bot.cs

```diff
@@ -50,6 +50,22 @@ public sealed class Bot {
 			AssertResultMatchesExpectation(expectedResult, itemsToSend);
 		}
 
+		[TestMethod]
+		public void TooManyCardsForSingleTrade() {
+			const uint appID = 42;
+
+			HashSet<Steam.Asset> items = new();
+
+			for (uint i = 0; i < ArchiSteamFarm.Trading.MaxItemsPerTrade; ++i) {
```

Comment:
Since `Trading.MaxItemsPerTrade` is `byte`, please change `i` to be `byte` as well. Also use `i++` instead to stay consistent with the ASF codebase.

Reply:
Yep. Had a `ArchiSteamFarm.Trading.MaxItemsPerTrade + 42` there earlier, which would have gone over `byte.MaxValue` but removed it since it's unnecessary anyways. Thanks for pointing it out.